tensorflow/core/kernels/fixed_length_record_reader_op.cc

```diff
@@ -107,11 +174,15 @@ class FixedLengthRecordReader : public ReaderBase {
   const int64 record_bytes_;
   const int64 footer_bytes_;
   const int64 hop_bytes_;
+  string lookahead_cache_;
   Env* const env_;
-  int64 file_pos_limit_;
   int64 record_number_;
-  std::unique_ptr<RandomAccessFile> file_;  // must outlive input_buffer_
-  std::unique_ptr<io::InputBuffer> input_buffer_;
+  string encoding_;
+  // must outlive buffered_input_stream_
+  std::unique_ptr<RandomAccessFile> file_;
+  // must outlive buffered_input_stream_
+  std::unique_ptr<io::RandomAccessInputStream> file_stream_;
+  std::unique_ptr<io::InputStreamInterface> buffered_input_stream_;
```

Comment:
May be call this `buffered_inputstream_` to be consistent with the file naming convention(buffered_inputstream.h etc).